ap.md

```diff
@@ -0,0 +1,228 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+<div class="large-8 columns">
+# Build Your Own Internet
+
+Along with connecting to an existing Wifi network, Tessel 2 can also act as a portable router and emit a custom Wifi network. We can use this functionality to connect directly with our Tessel through a web browser. By the end of this article, we'll be able to control the Tessel LEDs through a web app served by the Tessel.  
```

Comment:
So my assumption is that since Tessel is serving the network, latency between requests to the Tessel server and interactions on the served web page is reduced. It also means there is no dependency on an existing wifi network to connect with Tessel via the web. 

Does that sound like an accurate description?